internal/wasm/options.go

```diff
@@ -0,0 +1,108 @@
+// Copyright Envoy Gateway Authors
+// SPDX-License-Identifier: Apache-2.0
+// The full text of the Apache license is available in the LICENSE file at
+// the root of the repo.
+
+// Copyright Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package wasm
+
+import (
+	"time"
+
+	"k8s.io/apimachinery/pkg/util/sets"
+)
+
+const (
+	DefaultPurgeInterval         = 1 * time.Hour
+	DefaultModuleExpiry          = 24 * time.Hour
+	DefaultHTTPRequestTimeout    = 15 * time.Second
+	DefaultHTTPRequestMaxRetries = 5
+	DefaultPullTimeout           = 5 * time.Minute
+	DefaultMaxCacheSize          = 1024 * 1024 * 1024 * 10 // 10GB
```

Comment:
is this static or dynamic ? id suggest making this lesser, e.g. to 1GB

Reply:
It's the default value for the cached wasm file size. We can expose this and other cache configuration options to API if needed in the future.